### PR TITLE
raidboss: Initial r2s

### PR DESCRIPTION
**Pull request by valarnin**
Did some testing in zone but not extensive.

### CODE REVIEW

File: ui/raidboss/data/07-dt/raid/r2s.ts [hunk 1]
```diff
@@ -1,14 +1,226 @@
...
-  triggers: [],
+  initData: () => ({
+    partnersSpreadCounter: 0,
+  }),
+  triggers: [
+    {
+      id: 'R2S Headmarker Shared Tankbuster',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.sharedBuster, capture: true },
+      response: Responses.tankBuster(),
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/raid/r2s.ts:37]:
```suggestion
      response: Responses.sharedTankBuster(),
```

File: ui/raidboss/data/07-dt/raid/r2s.ts [hunk 2]
```diff
@@ -1,14 +1,226 @@
...
+    },
+    {
+      id: 'R2S Headmarker Cone Tankbuster',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.tankLaser, capture: true },
+      response: Responses.tankBuster(),
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/raid/r2s.ts:43]:
```suggestion
      response: Responses.tankCleave(),
```

**Comment by Legends0** [comment]
I think the Centerstage and Outerstage combo triggers could use a durationSeconds as the tts is still talking after the text expires. I used 9s.

Reducing the TTS to this also made it more understood to me:
Under Intercards => Out => Cards
Out Cards => Intercards => Under

The R2S Headmarker Spread call I think should just go to those with the marker and not everyone. There's multiple ways to do this strat though.

File: ui/raidboss/data/07-dt/raid/r2s.ts [hunk 3]
```diff
@@ -1,14 +1,226 @@
...
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R2S Honey B. Live: 2nd Beat',
+      type: 'StartsUsing',
+      netRegex: { id: '9C25', source: 'Honey B. Lovely', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R2S Honey B. Live: 3nd Beat',
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r2s.ts:178]:
```suggestion
      id: 'R2S Honey B. Live: 3rd Beat',
```

**Review by Legends0 — APPROVED** [review]

**Review by Echoring — APPROVED** [review]

**Comment by Battler624** [comment]
I haven't done this a lot but this is the second week in a row i notice this so i figured i'd report it.
I was using Gyoten while looking the the timeline of the poison sting (drop) and honestly its just plain wrong by around 3/4 of a second? I do not have exact numbers and idk how to provide them but everytime I dashed at the 0.0 mark I'd drop the puddle in the middle of the arena.

The one time I didn't drop it in the middle was when I had the second drop and I dashed when the third drop was at 4.3s left.

If there is anything I can do to help fix this, please do tell me I'd be happy to help.

**Pull request merged**